scripts/clear-tile-cache

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+BUCKET_NAME=$(docker-compose run --rm tilegarden bash -c 'echo -n $PFB_TILEGARDEN_CACHE_BUCKET')
```

Comment:
Do you think it's worth allowing the user to optionally set `BUCKET_NAME` if they want to clear a bucket other than the one that's configured for dev? e.g.:

```bash
if [[ -z "${BUCKET_NAME}" ]]; then
    BUCKET_NAME=$(docker-compose run --rm tilegarden bash -c 'echo -n $PFB_TILEGARDEN_CACHE_BUCKET')
fi
```

I agree with what you wrote in the PR body, that it seems like the usefulness of this script is probably limited to the isolated case where you want to clear the dev cache for testing -- still, I wonder if more extensibility will come in handy. I leave the call up to you.